content/en/docs/tasks/administer-cluster/limit-storage-consumption.md

```diff
@@ -28,9 +27,9 @@ how much storage a single namespace can consume in order to control cost.
 
 The admin would like to limit:
 
-1. The number of persistent volume claims in a namespace
-2. The amount of storage each claim can request
-3. The amount of cumulative storage the namespace can have
+1. The number of persistent volume claims in a namespace.
+2. The amount of storage each claim can request.
+3. The amount of cumulative storage the namespace can have.
```

Comment:
None of these items is a complete sentence, so the periods are no necessary.

Reply:
Thank.